build/Common.props

```diff
@@ -29,7 +29,7 @@
     <MicrosoftAspNetCoreHttpFeaturesPkgVer>[2.1.1,5.0)</MicrosoftAspNetCoreHttpFeaturesPkgVer>
     <MicrosoftAspNetTelemetryCorrelationPkgVer>[1.0.7,2.0)</MicrosoftAspNetTelemetryCorrelationPkgVer>
     <MicrosoftCodeAnalysisFxCopAnalyzersPkgVer>[3.3.0-beta2.final]</MicrosoftCodeAnalysisFxCopAnalyzersPkgVer>
-    <MicrosoftCodeCoveragePkgVer>[16.6.1]</MicrosoftCodeCoveragePkgVer>
+    <MicrosoftCodeCoveragePkgVer>[16.8.0-preview-20200812-03]</MicrosoftCodeCoveragePkgVer>
```

Comment:
any reason why we want to move to preview version?

Reply:
Some time ago (a week maybe), I tried the version `16.7.0`. But that version came with a bug, which I reported.
Now, they launched the preview and asked me to test again. If that works fine, they will release a stable one shortly.

So, in the end we will use the version `16.7.1`.